lib/read-stream.js

```diff
@@ -143,15 +156,17 @@ class ReadStream extends stream.Readable {
       this._readBlock()
     }
 
-  }
+  },
 
   destroy() {
     fs.close( this.fd, ( error ) => {
       this.fd = null
       stream.Readable.prototype.destroy.call( this )
     })
-  }
+  },
 
 }
 
+inherit( ReadStream, stream.Readable )
```

Comment:
Does this follow the same _discouraged_ behaviour as https://nodejs.org/docs/latest/api/util.html#util_util_inherits_constructor_superconstructor ?